pkg/controller/v1alpha2/applicationconfiguration/render.go

```diff
@@ -469,6 +477,17 @@ func fillValue(obj *unstructured.Unstructured, fs []string, val interface{}) err
 	return nil
 }
 
+func addWorkloadLabels(w *unstructured.Unstructured, labels map[string]string) {
```

Comment:
can you move this to util and change the name from "addWorkloadLabels" to "addLabels" as this is very generic. Also, I am not sure if this already exists somewhere in the library

Reply:
Yes, I will check and move it.